guides/micronaut-jms-aws-sqs/src/main/resources/logback.xml

```diff
@@ -0,0 +1,15 @@
+<configuration>
```

Comment:
delete this file - it's the same as the file that's generated by the `create-app` step during the build